core/src/main/java/org/mule/api/MuleMessage.java

```diff
@@ -573,4 +458,25 @@
      * {@link org.mule.api.transport.PropertyScope#OUTBOUND}.
      */
     void clearAttachments();
+
+    /**
+     * Update the value of the message originalPayload.  This is typically updating automatically but there are some
```

Comment:
If possible, always start the javadoc with a verb (update > updates).
Fix all javadoc formats